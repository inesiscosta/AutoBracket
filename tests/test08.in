50 50
11 17 6 4 14 23 13 44 44 8 36 45 50 6 35 23 30 4 12 44 13 18 43 21 35 10 21 7 27 43 4 26 29 13 25 31 12 14 36 16 40 24 11 46 19 44 8 36 29 14
39 37 25 21 19 2 46 49 11 40 6 36 4 27 27 13 20 2 36 6 21 34 3 3 6 21 50 47 15 22 28 46 45 1 45 44 19 17 40 33 24 35 30 6 39 39 45 25 30 49
9 31 19 4 15 19 15 41 18 37 3 47 37 34 18 12 20 12 34 20 37 11 25 49 16 20 22 11 29 24 32 48 11 15 10 29 50 28 41 38 9 7 18 33 4 24 1 5 11 7
28 1 25 43 25 12 44 49 20 23 33 7 31 22 36 28 48 33 21 47 7 24 30 41 42 12 26 42 23 22 23 34 5 24 49 39 45 30 21 26 19 19 48 7 12 31 3 35 22 24
23 9 36 11 7 3 49 27 29 3 16 21 34 4 50 42 40 14 8 29 28 19 16 46 25 13 25 6 27 37 32 25 37 40 25 26 47 32 27 3 48 33 14 28 30 32 37 43 35 50
14 39 14 45 44 46 18 5 42 45 49 29 8 7 8 3 46 36 49 39 29 17 6 30 30 32 47 35 39 10 9 14 26 3 15 41 19 37 14 7 41 1 42 30 5 14 11 43 14 12
11 49 31 17 37 34 20 35 18 29 46 32 26 34 46 21 45 40 45 44 46 1 6 33 28 40 9 45 34 24 8 21 26 25 33 8 8 31 38 48 48 26 24 26 47 13 12 49 19 36
47 28 46 15 28 15 37 31 5 33 1 22 34 44 28 32 13 2 43 26 14 40 13 41 36 50 44 28 29 46 24 29 15 32 14 22 27 23 46 18 41 18 44 46 47 17 43 48 29 41
36 41 29 31 12 45 1 21 31 5 7 47 3 27 8 39 10 40 3 19 18 1 11 33 30 49 2 22 40 27 6 44 6 44 4 29 34 9 21 17 25 49 15 37 18 29 11 50 5 4
32 19 18 46 49 28 37 6 46 44 44 36 39 28 40 36 45 3 5 38 1 32 17 50 34 21 31 15 11 49 11 36 40 23 41 29 32 4 37 39 12 41 31 1 26 34 15 39 7 32
48 6 28 13 16 29 47 24 18 49 4 23 31 43 30 24 43 17 5 6 12 26 8 9 15 30 34 4 27 20 7 28 37 8 24 5 24 44 25 37 40 45 24 11 29 17 37 14 2 22
30 42 50 31 4 44 24 44 19 46 40 16 38 46 17 46 25 40 26 23 32 37 49 2 32 36 33 41 8 35 10 9 45 1 39 34 28 20 28 19 43 1 9 20 42 35 14 36 3 45
38 33 23 43 17 37 10 16 20 25 2 23 17 12 36 2 22 19 43 28 18 39 25 47 20 26 17 42 7 50 37 24 31 13 50 39 24 35 25 34 1 29 37 20 43 28 46 49 41 24
38 37 16 23 29 37 50 14 47 49 41 16 22 43 28 44 27 45 35 49 8 19 42 31 5 11 39 13 34 34 24 38 23 10 33 44 22 12 26 45 22 39 20 29 43 44 5 12 1 34
11 39 1 39 35 3 23 5 1 32 8 18 41 10 18 2 8 45 42 7 9 44 21 43 34 42 25 1 1 40 31 33 21 49 8 41 2 47 43 22 40 47 21 4 48 3 4 3 33 37
41 8 25 41 5 7 17 35 16 28 31 23 33 39 47 31 1 18 13 33 26 33 44 46 30 41 33 30 11 13 16 38 31 35 37 31 42 25 6 26 11 8 15 39 3 21 18 41 21 27
28 7 5 40 42 29 46 48 25 1 9 49 40 35 43 26 26 10 13 39 28 47 28 3 22 33 26 43 23 13 42 39 39 15 18 18 6 18 6 12 40 49 20 46 24 50 13 46 37 25
3 10 6 37 22 50 31 28 44 38 34 39 26 49 23 29 49 48 15 27 9 27 19 47 1 29 35 24 47 26 27 12 16 50 40 4 3 40 41 33 8 22 36 3 5 34 28 13 6 27
11 32 1 30 23 27 33 45 18 16 20 39 38 38 38 17 11 27 15 1 45 49 16 49 18 18 44 21 21 2 19 26 24 4 2 12 9 44 16 13 32 46 50 40 24 34 1 27 20 9
13 39 40 26 20 39 22 22 12 31 43 17 5 24 20 48 10 50 13 12 22 50 37 37 28 20 38 20 32 44 46 31 13 30 12 34 30 24 24 23 37 3 11 21 41 9 34 20 13 29
40 25 37 40 46 15 5 26 31 35 44 3 37 25 14 8 43 30 47 20 46 11 40 12 14 20 29 31 33 44 39 31 10 33 8 1 37 12 19 23 35 40 25 7 43 7 21 34 10 5
8 13 40 15 33 17 16 45 34 14 26 5 20 33 1 21 21 28 38 28 24 34 36 10 16 43 24 32 29 39 2 38 28 26 48 47 48 30 13 30 5 14 22 25 11 5 18 9 44 31
1 25 35 9 22 21 25 27 1 39 39 30 21 27 13 18 42 31 48 14 9 40 23 38 8 26 30 6 43 23 19 32 16 50 35 20 41 7 26 50 7 3 50 49 8 38 25 48 9 14
8 47 7 5 41 33 29 10 3 37 40 34 36 9 26 48 40 16 42 40 21 42 36 48 26 50 5 43 39 14 23 41 9 16 27 21 32 46 3 26 36 9 2 28 2 49 39 4 17 6
44 41 25 25 24 41 37 14 43 36 7 50 13 21 28 49 27 3 44 43 32 24 13 9 36 5 1 2 16 37 48 34 24 13 31 25 24 30 49 25 40 34 6 17 45 43 33 25 22 34
38 38 8 47 33 27 16 30 29 31 1 16 14 13 23 44 30 47 41 26 16 41 30 3 39 35 43 41 20 1 5 49 33 43 28 45 15 18 10 31 32 12 26 44 24 15 42 19 43 9
34 13 48 40 35 25 42 34 12 17 41 2 6 4 28 3 11 34 36 46 47 40 13 14 19 37 12 38 32 14 44 18 48 48 7 23 15 22 48 20 7 11 46 32 13 28 27 17 24 36
40 26 19 9 45 8 12 6 2 34 46 45 19 36 5 42 47 46 20 23 40 11 35 31 14 22 36 36 35 18 4 18 12 39 40 27 1 49 3 32 24 8 31 10 30 5 34 17 26 7
14 20 14 39 9 24 46 15 36 1 11 17 6 13 39 14 48 32 31 18 43 50 10 35 15 21 5 5 22 35 41 14 47 46 24 49 14 16 47 31 20 31 26 17 34 11 6 26 41 43
38 13 29 32 36 34 30 15 10 35 49 29 25 43 29 9 33 22 11 22 26 30 23 49 23 23 50 16 39 34 35 8 8 17 50 6 23 13 35 45 26 24 23 48 4 29 13 47 14 11
40 18 36 30 36 4 46 11 4 28 9 25 14 39 1 25 36 1 3 41 27 35 28 3 46 5 17 30 50 21 10 37 11 30 32 49 4 32 6 41 17 49 39 21 20 44 35 44 25 12
3 1 45 21 40 35 17 28 28 46 10 18 8 15 30 11 21 18 40 6 16 43 46 42 27 39 46 20 2 40 9 40 7 3 45 42 13 28 16 47 20 33 33 19 33 32 24 38 7 48
9 10 19 44 16 18 48 17 41 38 41 35 20 44 46 3 15 28 2 37 33 16 22 46 42 3 45 40 8 21 22 14 10 12 10 19 16 46 15 18 49 3 5 40 11 41 34 44 49 13
45 14 23 35 18 16 34 46 9 6 42 32 29 47 3 10 22 2 20 43 21 6 6 44 42 2 33 43 27 10 49 6 38 9 19 6 40 33 15 17 31 38 27 35 34 25 20 21 41 33
4 47 33 24 39 29 46 12 7 38 38 47 45 16 16 34 18 38 38 2 18 47 49 36 36 32 29 36 44 20 13 16 46 27 25 29 20 44 20 16 19 16 43 13 13 40 34 26 13 38
44 13 11 8 18 1 47 44 14 5 33 39 40 12 5 37 40 16 44 46 46 48 3 39 9 14 37 41 7 19 17 34 42 16 15 18 16 41 22 22 20 50 21 15 18 47 12 41 45 24
35 33 27 50 30 40 26 44 33 5 48 7 21 26 2 24 23 6 16 32 8 44 25 24 4 46 43 12 41 34 46 49 3 34 7 27 20 9 16 27 4 19 2 32 17 35 39 31 49 37
36 3 22 26 43 26 3 14 42 1 14 6 37 17 37 11 8 35 27 15 25 46 11 45 36 42 47 29 29 26 16 35 20 5 48 27 40 50 39 19 17 32 36 7 2 29 9 7 13 8
4 24 10 32 38 9 3 13 14 4 10 38 11 35 19 1 26 32 42 10 4 22 35 21 32 41 35 15 28 37 46 49 33 17 49 6 38 48 9 22 26 14 29 15 24 42 29 26 2 22
42 43 11 1 42 27 35 39 7 17 43 21 40 39 7 47 3 45 46 25 38 37 23 35 34 40 21 35 47 15 28 25 46 20 40 28 40 20 28 4 27 37 14 22 4 5 24 50 32 45
43 26 22 17 9 22 16 15 41 8 37 13 31 49 27 24 35 6 7 42 47 41 33 2 9 13 36 10 45 30 20 13 38 47 27 45 38 26 29 4 32 37 38 4 9 14 34 50 30 1
36 44 27 35 33 9 5 4 23 41 40 34 33 38 20 28 25 38 43 8 19 28 16 33 36 30 27 36 49 5 39 18 9 34 31 40 36 40 34 38 30 21 1 49 33 33 7 4 36 27
17 30 41 12 43 47 15 38 41 32 41 36 42 47 32 6 18 35 34 37 2 38 3 38 24 1 47 21 34 2 7 31 40 22 25 42 50 3 31 29 29 30 25 35 27 19 22 4 30 45
36 41 47 38 2 17 48 41 5 22 1 26 28 23 5 31 14 43 20 14 12 8 25 24 15 46 40 29 40 45 13 27 2 24 24 6 37 49 3 12 17 5 48 13 18 46 19 26 4 17
11 20 6 25 8 32 31 12 43 1 23 33 41 14 47 27 23 45 4 22 29 37 11 42 47 21 46 39 30 6 32 7 19 4 20 33 6 37 41 28 15 34 45 33 43 49 40 41 43 9
25 4 26 36 21 45 1 17 30 41 45 33 22 2 5 50 17 42 20 8 11 34 30 37 47 15 13 38 40 49 6 42 19 9 2 18 31 43 22 19 28 19 22 7 40 3 3 3 42 5
26 8 11 28 45 6 14 5 10 34 18 3 4 3 11 7 2 8 40 38 9 24 16 12 18 10 29 48 46 32 46 18 34 25 21 44 23 17 25 21 29 6 9 22 45 11 45 17 39 1
32 43 18 16 18 24 43 17 14 20 15 35 6 13 13 29 46 16 9 28 33 28 25 2 22 14 28 27 20 29 24 14 34 29 6 12 38 10 4 20 5 29 10 8 17 32 23 2 46 18
5 3 2 7 9 33 24 39 49 47 21 8 37 10 40 30 13 39 5 37 33 39 15 4 41 34 25 26 29 43 36 42 1 32 39 50 3 20 21 8 38 33 46 13 21 35 36 42 32 17
1 2 4 31 24 8 39 32 29 10 6 23 49 27 5 24 35 43 1 4 16 34 47 11 23 41 2 4 7 22 41 28 50 24 43 49 18 33 2 49 38 7 1 23 16 41 46 30 14 47
46 41 18 41 43 14 43 13 46 17 46 41 19 11 17 13 29 23 11 36 39 49 20 29 1 17 24 37 25 22 33 31 8 21 24 31 11 45 31 46 11 47 19 31 36 5 17 18 20 16
32
