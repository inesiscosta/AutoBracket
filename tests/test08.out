1
((((((((((((((((((((((((((((((((((((((((((46 41) 18) 41) 43) 14) 43) 13) 46) 17) 46) 41) 19) 11) 17) 13) 29) 23) 11) 36) 39) 49) 20) (29 1)) 17) 24) (37 25)) 22) 33) 31) 8) ((21 24) 31)) 11) ((45 31) 46)) 11) 47) 19) 31) (36 5)) 17) 18) 20) 16)
